core/core_test.go

```diff
@@ -57,8 +57,8 @@ func TestForwardToLeader(t *testing.T) {
 		httpClient: &http.Client{
 			Transport: &http.Transport{
 				TLSClientConfig: &tls.Config{
-					RootCAs: certpool,
-				},
+					RootCAs:    certpool,
```

Comment:
Per offline conversation, this is a simulated Chain Core API http object, not an external client object connecting to a Chain Core.